frontend/src/util/CommentState.js

```diff
@@ -8,4 +8,9 @@ export const commentThreadsState = atomFamily({
 export const commentThreadIDsState = atom({
   key: "commentThreadIDs",
   default: new Set([]),
+});
```

Comment:
don't htink this file is used anywhere else in this PR? so maybe pull it out of this one